gui-apps/waybar/waybar-0.9.2.ebuild

```diff
@@ -0,0 +1,60 @@
+# Copyright 2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit meson
+
+DESCRIPTION="Highly customizable Wayland bar for Sway and Wlroots based compositors."
+HOMEPAGE="https://github.com/Alexays/Waybar"
+
+if [[ ${PV} == 9999 ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/Alexays/${PN^}.git"
+else
+	SRC_URI="https://github.com/Alexays/${PN^}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64"
+fi
+
+LICENSE="MIT"
+SLOT="0"
+IUSE="mpd network pulseaudio tray +udev"
+
+BDEPEND="
+	>=app-text/scdoc-1.9.2
+	virtual/pkgconfig
+"
+
+DEPEND="
+	dev-cpp/gtkmm:3.0
+	dev-libs/jsoncpp:=
+	dev-libs/libinput:=
+	dev-libs/libsigc++:2
+	>=dev-libs/libfmt-5.3.0:=
+	>=dev-libs/spdlog-1.3.1:=
+	dev-libs/wayland
+	dev-libs/wayland-protocols
```

Comment:
It should depend on https://github.com/HowardHinnant/date IIRC. I'm not entirely sure how it handles fallback, when this library is missing, but I'd assume, since it tries to link to it from system, it should be added as well? link https://github.com/Alexays/Waybar/blob/4b2e6b54a7fcfa5aae2bbc06f8683bbcf3855849/meson.build#L101

Reply:
is this even there in gentoo ?

Reply:
no, it should be packaged as well, I guess. Also you should likely link the https://bugs.gentoo.org/712236 bug.

Reply:
Hmmm, I don't think its a good idea to use this. The library has been superseded by the `std::chrono` directive in C++ 20 and is now on its last release. 
I will open an issue at waybar to talk about this.

Reply:
actually seems like: https://github.com/Alexays/Waybar/issues/668